src/app/modules/time-clock/components/entry-fields/entry-fields.component.html

```diff
@@ -7,6 +7,7 @@
       formControlName="activity_id"
       [class.is-invalid]="activity_id.invalid && activity_id.touched"
       required>
+      <option value="" selected="selected"></option>
```

Comment:
I think this change is causing this behavior at the time of displaying the activities:
![image](https://user-images.githubusercontent.com/56373098/123847432-334cad80-d8dc-11eb-9ad4-07ea0a990b61.png)
I think from my point of view an empty option should not be displayed.